### PR TITLE
T15504 psr 1.1

### LINKED ISSUE #15504: [NFR]: Support of PHP-PSR 1.1.0
**Is your feature request related to a problem? Please describe.**
Docs says that we are required to use `php-psr 1.0.*`.
> NOTE: You will need the PSR 1.0 extension installed. 

But this is a problem if other frameworks are used at the same time like Symfony, which in new versions requires `psr/container ^1.1`. The PHP-PSR 1.0.x specifies version `1.0.0` for `psr/container`. Because of this, we are in an incomprehensible situation when a project requires `psr/container ^1.1` in the composer.json, successfully installs it, and in fact, `psr/container` from PHP-PSR extension is used, since it has priority over the composer's library. 

**Describe the solution you'd like**
Allow to install [PHP-PSR extension 1.1.0](https://github.com/jbboehr/php-psr/releases/tag/v1.1.0)

**Additional context**
According to the PHP-PSR changelog, the difference between version 1.0.1 and version 1.1.0 is only in fixing support for `psr/container` to version `1.1.1`.

Also I have installed the PSR extension 1.1.0 locally and don't see any problems with my project. Maybe this only needs to be fixed in the documentation?

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: bug fix
* Link to issue: #15504 

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Upgraded PSR to 1.1.x and corrected `Phalcon\Container` interface

Thanks

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
         "ext-mbstring": "*",
         "ext-msgpack": "*",
         "ext-pdo": "*",
-        "ext-psr": "^0.7|^1.0",
+        "ext-psr": "^^1.1",
```
Comment (Jeckerson) [inline composer.json:49]:
```suggestion
        "ext-psr": "^1.1",
```

**Comment by niden** [comment]
Converting to draft and putting it on hold until pecl releases psr 1.1.1 (with container 1.1.1)

**Comment by StudioMaX** [comment]
`psr/container 1.1.1` is already included in `PHP-PSR 1.1.0` - https://pecl.php.net/package/psr/1.1.0
> - Update psr/container to 1.1.1

**Comment by Jeckerson** [comment]
Blocked by https://github.com/phalcon/cphalcon/pull/15610

**Review by Jeckerson — APPROVED** [review]

**Pull request merged**